server/i18n-bootstrap/lang-revisions.json

```diff
@@ -0,0 +1 @@
+{"af":"34825","am":"33757","an":"33758","ar":"35574","as":"33759","ast":"33760","az":"34206","bel":"33762","bg":"33707","bn":"33708","bo":"34091","br":"33709","bs":"34302","ca":"34733","ckb":"33710","cs":"34200","cy":"34948","da":"34726","de":"35575","de_formal":"35576","dv":"34829","el":"34207","el-po":"35396","en-gb":"34305","eo":"33716","es":"35577","es-cl":"34158","es-mx":"35578","et":"34982","eu":"33863","fa":"35572","fi":"34953","fo":"33720","fr":"35579","fr-be":"34456","fr-ca":"34735","fr-ch":"34457","fy":"18327","ga":"33724","gd":"33725","gl":"35011","gu":"33773","he":"35580","hi":"35262","hr":"34824","hu":"34939","hy":"33971","id":"35581","is":"34729","it":"35582","ja":"35583","ka":"33727","kir":"33728","kk":"33729","km":"33730","kn":"33777","ko":"35584","la":"18327","lo":"33731","lt":"34730","lv":"33733","mk":"33736","ml":"33737","mn":"35231","mr":"33738","ms":"33739","mt":"18327","mwl":"33740","ne":"33780","nl":"35585","nn":"34155","no":"35527","oci":"33743","pa":"35562","pl":"35501","ps":"33744","pt":"35160","pt-br":"35586","ro":"35571","ru":"35587","rup":"33785","si":"33745","sk":"33746","sl":"33747","snd":"33787","so":"33748","sq":"35395","sr":"34740","sr_latin":"34744","su":"33749","sv":"35573","ta":"34572","te":"35476","th":"33704","tir":"18327","tl":"33752","tr":"35588","ug":"33753","uk":"34000","ur":"34096","uz":"33793","vi":"34741","yi":"18327","yo":"18327","zh-cn":"35589","zh-tw":"35590"}
```

Comment:
(I agree with what you said in the PR desc -- this shouldn't really live in this git repo.)